packages/components/app/styles/components/side-nav.scss

```diff
@@ -0,0 +1,259 @@
+//
+// SIDE-NAV
+//
+
+@use "../mixins/focus-ring" as *;
+
+// WIP, naming, etc. will be refined... TODO: Add JSON for color tokens, etc.
+:root {
+  // Sizes:
+
+  // header, body, footer "wrappers":
+  --token-side-nav-wrapper-padding-horizontal: 16px;
+  --token-side-nav-wrapper-padding-vertical: 16px;
+
+  // header content:
+  --token-side-nav-header-home-link-padding: 4px;
+  --token-side-nav-header-home-link-logo-size: 48px;
+  --token-side-nav-header-actions-spacing: 12px;
+
+  // body content:
+  --token-side-nav-body-list-margin-vertical: 24px;
+  --token-side-nav-body-list-item-padding-horizontal: 8px;
+  --token-side-nav-body-list-item-padding-vertical: 4px;
+  --token-side-nav-body-list-item-content-spacing-horizontal: 8px;
+  --token-side-nav-body-list-item-border-radius: 5px;
+
+  // Colors:
+
+  // SideNav container:
+  --token-side-nav-foreground-primary: #dedfe3;
+  --token-side-nav-foreground-primary-strong: #fff;
+  --token-side-nav-foreground-faint: #8c909c;
+  --token-side-nav-surface: #0c0c0e;
+
+  // Interactive elements:
+  // base:
+  --token-side-nav-interactive-surface-hover: #3b3d45;
+  --token-side-nav-interactive-surface-active: #656a76;
+
+  // selected / current:
+  --token-side-nav-interactive-selected-surface-default: #656a76;
+  --token-side-nav-interactive-selected-surface-hover: #3b3d45;
+  --token-side-nav-interactive-selected-surface-active: #656a76;
+}
```

Comment:
@cveigt Above are the tokens I defined if you would like to review. Probably the most relevant ones for you are the colors.

Reply:
@didoo @cveigt I'd like to confirm with you both that these tokens are what we want. Please let me know also if there are any others I should add.

After these are confirmed I will create the JSON file for them.

Reply:
@KristinLBradley added a couple of comments, but nothing that would block the release (we can always update the names later, if needed)